src/onyx/messaging/aeron.clj

```diff
@@ -17,6 +17,38 @@
              [java.util.function Consumer]
              [java.util.concurrent TimeUnit]))
 
+(def max-port 50000)
```

Comment:
I meant to come back to that. Though honestly it's only really used for the range. I mean we die if we run out of ports anyway, right? It may be better to just pick a maximum number of peers we assume can ever run and add it on to the start port? If we ever hit that number we abort allocation - it's probably because peers aren't ckeaning up properly anyway?